src/components/Tool/FixPhoneNumbers/helper.test.ts

```diff
@@ -0,0 +1,137 @@
+import { determineBulkDataToSend } from './helper';
+
+const testData = [
+  [
+    {
+      id: 'testid1',
+      firstName: 'Test',
+      lastName: 'Contact',
+      avatar: 'https://www.example.com',
+      isNewPhoneNumber: false,
+      newPhoneNumber: '',
+      phoneNumbers: {
+        nodes: [
+          {
+            id: 'id1',
+            updatedAt: new Date('2021-06-21T03:40:05-06:00').toISOString(),
+            number: '+3533895891',
+            primary: true,
+            source: 'MPDX',
+          },
+          {
+            id: 'id2',
+            updatedAt: new Date('2021-06-21T03:40:05-06:00').toISOString(),
+            number: '3533895892',
+            primary: false,
+            source: 'MPDX',
+          },
+          {
+            id: 'id3',
+            updatedAt: new Date('2021-06-21T03:40:05-06:00').toISOString(),
+            number: '+623533895893',
+            primary: false,
+            source: 'MPDX',
+          },
+        ],
+      },
+    },
+  ],
+  [
+    {
+      id: 'testid2',
+      firstName: 'Test',
+      lastName: 'Contact',
+      avatar: 'https://www.example.com',
+      isNewPhoneNumber: false,
+      newPhoneNumber: '',
+      phoneNumbers: {
+        nodes: [
+          {
+            id: 'id1',
+            updatedAt: new Date('2021-06-21T03:40:05-06:00').toISOString(),
+            number: '+3533895894',
+            primary: true,
+            source: 'DataServer',
+          },
+          {
+            id: 'id2',
+            updatedAt: new Date('2021-06-21T03:40:05-06:00').toISOString(),
+            number: '3533895895',
+            primary: false,
+            source: 'DataServer',
+          },
+          {
+            id: 'id3',
+            updatedAt: new Date('2021-06-21T03:40:05-06:00').toISOString(),
+            number: '+623533895896',
+            primary: false,
+            source: 'MPDX',
+          },
+        ],
+      },
+    },
+  ],
+];
+
+describe('FixPhoneNumbers-helper', () => {
+  it('Should return a Contact with all phoneNumbers', async () => {
```

Comment:
Since these tests don't contain an `await` you can remove the `async` from each of them.

```suggestion
  it('Should return a Contact with all phoneNumbers', () => {
```